src/status_im2/subs/wallet/networks.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.subs.wallet.networks
+  (:require [quo.foundations.resources :as resources]
+            [re-frame.core :as re-frame]
+            [status-im2.constants :as constants]))
+
+(re-frame/reg-sub
+ :wallet/filtered-networks-by-mode
+ :<- [:wallet/networks]
+ (fn [networks [_ test?]]
+   (get networks (if test? :test :prod))))
+
+(def ^:const mainnet-network-details
+  {:source       (resources/networks :ethereum)
+   :short-name   "eth"
+   :network-name :ethereum})
+
+(def ^:const arbitrum-network-details
```

Comment:
Would be nice to have a `contexts/wallet/contants.cljs` file. I'm sure it'll be useful to isolate other wallet related constants.

Reply:
sure, I can move these details to there. The problem is that some of the data in status go is slightly incorrect for these details - however most such as the urls etc are correct. But sure I will put these in the constants. at least for the details such as `"eth" `:etheruem` etc